socorro/unittest/external/es/base.py

```diff
@@ -774,6 +794,15 @@ def test_with_version(self):
     return decorated
 
 
+class SuperSearchWithFields(SuperSearch):
```

Comment:
Oh. I see. You just moved this from elsewhere. Anyway, I think it's always good to keep things simple because it's easier to debug and refactor. But you're free to keep it.